tk/tk.go

```diff
@@ -93,6 +93,20 @@ func TkLibrary() (path string) {
 	return
 }
 
+func AutoPath() (path string) {
+	path, _ = evalAsString("puts $auto_path")
+	return
+}
+
+func SetAutoPath(path string) string {
+	r, _ := evalAsString(fmt.Sprintf("set auto_path [linsert $auto_path 0 %s]", path))
+	return r
+}
```

Comment:
rm these. there is a separate PR. merge that in to master then update this branch with master